ecmascript/transforms/src/typescript/strip.rs

```diff
@@ -713,26 +714,23 @@ impl VisitMut for Strip {
                     .scope
                     .imported_idents
                     .get(&(local.sym.clone(), local.span.ctxt()));
-                match self.config.import_not_used_as_values {
-                    ImportNotUsedAsValues::Remove => match entry {
-                        Some(&DeclInfo {
-                            has_concrete: false,
-                            ..
-                        }) => false,
-                        _ => true,
-                    },
-                    ImportNotUsedAsValues::Preserve => match entry {
-                        Some(&DeclInfo {
-                            has_type: true,
-                            has_concrete: false,
-                            ..
-                        }) => false,
-                        _ => true,
-                    },
+                match entry {
```

Comment:
Is this safe? 
It seems like semantic is changed from current behavior to `remove`

Reply:
This is the actual fix. The semantics for this bit should always be "remove". The meaning of `importsNotUsedAsValues` in `tsc` is to do with whether to keep an only-type import as a side-effect import, or remove it.

Reply:
> Is this safe?

Always removing completely unused import identifiers won't cause any breakages, it's safe.

Reply:
You are right. Thanks!